rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2021-10-29"
+components = [ "rustfmt" ]
```

Comment:
do we want clippy in there?

Reply:
I seem to have clippy by default so looks like we don't need to explicitly include it.